pkg/config/feature_gates.go

```diff
@@ -14,6 +14,7 @@ const (
 	WeightedTargetGroups        Feature = "WeightedTargetGroups"
 	ServiceTypeLoadBalancerOnly Feature = "ServiceTypeLoadBalancerOnly"
 	EndpointsFailOpen           Feature = "EndpointsFailOpen"
+	DisableAWSVpcCNI            Feature = "DisableAWSVpcCNI"
```

Comment:
Doesn't it sound a little misleading? Like if ALBC would have a power to disable the CNI.
Also, it mentions the CNI plugin name, I didn't see any other CNI plugins which would have the directly accessible POD IPs but maybe there are some in the wild (or will come).

How about `EnableIPTargetType` similar the service controller feature gate from another PR?

Reply:
I agree the name sounds misleading. Basically, we wanted to introduce a flag that toggles all features dependent on AWS VPC CNI, currently it's just `ip` target-type. How about `AWSVpcCNIAvailable`?

Reply:
I think `EnableIPTargetType` is a more appropriate name. That fact that it is dependent on the VPC CNI being available is an internal detail.